cypress/integration/login.js

```diff
@@ -1,47 +1,53 @@
+import LoginPageVariables from '../obj/LoginPage'
+import CalendarPage from '../obj/CalendarPage'
+import TimeLoggingPage from '../obj/TimeLoggingPage'
+
+const loginVariables= new LoginPageVariables()
+const calendarVariables= new CalendarPage()
+const timeLoggingVariables= new TimeLoggingPage()
 describe('Login functionality', function() {
     
     it('Should display validation for empty user after attempted loggin', function () {
-        cy.visit('/')
-        cy.get('.Select.not-valid').should('not.visible')
-        cy.get('[type="submit"]').click()
-        cy.get('.Select.not-valid').should('be.visible')
+        loginVariables.rootPage()
+        loginVariables.getUserValidationIndicator().should('not.visible')
+        loginVariables.submit().click()
+        loginVariables.getUserValidationIndicator().should('be.visible')
     })
 
     it('Should be able to login with role User', function () {
         const date = new Date()
-        cy.get('[id="loginForm.userId"]').click({force:true})
-        cy.get('[aria-label="TestCon User 10"]').click()
-        cy.get('[id="loginForm.role"]').click({force:true})
-        cy.get('[aria-label="Team Lead"]').click()
-        cy.get('[type="submit"]').click()
-
-        cy.url().should('include', '/time-logging')
-        cy.get('.page__title').contains('Timesheets')
-        cy.get('.calendar').should('be.visible')
-        cy.get('.tile.form').should('be.visible')
-        cy.get('.user-info__title').contains('TestCon User 10')
-        cy.get('.main-nav').find('li').should('have.length', 2)
 
-        cy.get('.calendar__body').should('be.visible')
-        cy.get('.calendar--selected').contains(date.getDate())
+        loginVariables.loginFormIdList().click({force:true})
```

Comment:
If you are naming this variable it would be better to come up with something like "getLoginUserDreopdown"
also if a function returns something it is good practice to start its name with "get..."